### ISSUE TITLE
Dialog: Keyboard navigation

### ISSUE THREAD

**Issue by gyoshev** [telerik/kendo-angular#530]
The Dialog should be closable via the `Esc` key and trigger its primary action with `Enter`, if it is focused and if it contains no input elements.

**Comment by gyoshev** [comment]
A workaround for this is to attach a keyboard event handler and handle events accordingly, as shown in this [plunker snippet](http://plnkr.co/edit/F98J11QNhxceiXGGnOqY?p=preview).

**Comment by Raisolution** [comment]
Implemented in v3.2.0

**Comment by Raisolution** [comment]
HI @jmaronge,

Currently the Dialog keyboard navigation cannot be toggled.
